logistics/scripts/modules/postmanTestHarnessConfiguration.json

```diff
@@ -11,14 +11,14 @@
           "ApiClients": [
             {
               "ApiClientName": "255901",
-              "LocalEducationOrganizations": [255901],
+              "LocalEducationOrganizations": [ 255901 ],
```

Comment:
What tool did you use to format this file? It has introduced a lot of whitespace changes which create unnecessary noise in the history. All these whitespace changes should probably be reverted now (and not staged for the commit).

Reply:
I fixed whitespace issue in this file . Thanks